packages/knip/test/cli-reporter-markdown.test.ts

```diff
@@ -0,0 +1,31 @@
+import assert from 'node:assert/strict';
+import test from 'node:test';
+import { resolve } from '../src/util/path.js';
+import { execFactory } from './helpers/exec.js';
+
+const cwd = resolve('fixtures/module-resolution-non-std');
+
+const exec = execFactory(cwd);
+
+test('knip --reporter markdown', () => {
+  const markdown = `## Unused files (1)
+
+* src/unused.ts
+
+## Unlisted dependencies (2)
+
+| Symbol          | Location     |
```

Comment:
I know "symbol" is the internal key name from the results provided to the reporter, but this wasn't the best idea from my end, trying to come up with a single name for "identifier" for exports, "specifier" for unresolved imports, etc.

I guess either be correct for each issue type separately, or simply use "name" consistently for end-users in each issue type is maybe even better. WDYT?

Reply:
I think name is common for all types, let's use that?

Reply:
Alright, let's use "name" everywhere (we can always be more specific later on if necessary).